.github/workflows/nolints.yml

```diff
@@ -71,6 +76,7 @@ jobs:
           git config --unset http.https://github.com/.extraheader
 
       - name: file a new PR to update nolints.json and style-exceptions.txt
+        shell: bash
```

Comment:
Is this needed? (It probably doesn't hurt either, though.)

Reply:
I've removed it to minimize the diff.